package.json

```diff
@@ -185,7 +185,7 @@
     "yarn": "^1.3"
```

Comment:
update this file to point to the new `lbry-redux` commit
`5fb6efae24205dfdfa133ca4f26bccce033570e1`

Reply:
you can update `lbryinc` too since I just merged that PR.

`43d382d9b74d396a581a74d87e4c53105e04f845`